new-backend/server/apis/v2/services/informative.service.js

```diff
@@ -15,15 +15,36 @@ class InformativeService {
    * @returns {object} JSON representation of document
    * @memberof InformativeService
    */
-  async getByName(file) {
+  async getByName(folder, file) {
     try {
-      file += ".json";
-      // Open file containing our store
-      const pathToFile = path.join(process.cwd(), "App_Data/documents", file);
-      const text = await fs.promises.readFile(pathToFile, "utf-8");
-      // Parse the file content so we get an object
-      const json = await JSON.parse(text);
-      return json;
+      if (folder) {
+        file += ".json";
+        // Open file containing our store
+        const pathToFile = path.join(
+          process.cwd(),
+          "App_Data",
+          "documents",
+          folder,
+          file
+        );
+        const text = await fs.promises.readFile(pathToFile, "utf-8");
+        // Parse the file content so we get an object
+        const json = await JSON.parse(text);
+        return json;
+      } else {
+        file += ".json";
+        // Open file containing our store
+        const pathToFile = path.join(
+          process.cwd(),
+          "App_Data",
+          "documents",
+          file
+        );
+        const text = await fs.promises.readFile(pathToFile, "utf-8");
+        // Parse the file content so we get an object
+        const json = await JSON.parse(text);
+        return json;
+      }
```

Comment:
Hi guys. You seem to duplicate a lot of code, where the only difference is whether you join with the `folder` or not. Can you refactor?